test-suite/js/src/test/scala/org/scalajs/testsuite/library/ObjectTest.scala

```diff
@@ -0,0 +1,107 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.library
+
+import org.junit.Assert._
+import org.junit.Test
+import org.scalajs.testsuite.utils.Platform
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.{ JSBracketAccess, JSName }
+
+class ObjectTest {
+  import ObjectTest._
+
+  @Test def getOwnPropertyDescriptors(): Unit = {
+    val obj1 = new js.Object {
+      val property1 = 42
+    }
+    val propertyDescriptors = js.Object.getOwnPropertyDescriptors(obj1)
+    val property = propertyDescriptors("property1")
+    assertTrue(property.writable)
+    assertEquals(property.value, 42)
+  }
+
+  /* These need to be lazy vals, so that they do not blow up if there is no
```

Comment:
The comment does not seem seem to match the declarations. They are non-lazy vals at the moment.

Reply:
Removed in https://github.com/scala-js/scala-js/pull/3869/commits/a82fd73435cc98dc6a40374abab8ceb683da2d79